source/ftrack_user_location/configure_logging.py

```
@@ -29,31 +30,37 @@ def get_log_directory():
     return log_directory
 
 
-def configure_logging(logger_name, level=None, format=None, extra_modules=None):
-    '''Configure `loggerName` loggers with console and file handler.
+def configure_logging(
+    logger_name: str,
+    level: Optional[int] = None,
+    format: Optional[str] = None,
```

Ruff (0.15.15)

[error] Function argument `format` is shadowing a Python builtin

(A002)